css/index.css

```
@@ -162,12 +170,13 @@ h1{
 }
 
 .YU-signup{
-    top: 25%;
+    /*top: 25%;*/
+    top: 15.5%;
```

```suggestion
    top: 14%;
```
let's give it a bit more room